openwisp_monitoring/device/migrations/0002_create_template.py

```diff
@@ -0,0 +1,101 @@
+# Generated by Django 3.1.2 on 2020-12-20 06:15
+
+from collections import OrderedDict
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def migrate_data(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    if Template.objects.filter(
+        Q(config__contains='/usr/sbin/openwisp-monitoring')
+        & Q(config__contains='/usr/sbin/netjson-monitoring'),
+        default=True,
+    ).count():
+        return
+    default_template = Template(
```

Comment:
can you check if it's possible somehow to force a specific UUID?
If we can force the UUID, then we can automatically upgrade the template as well and document this behaviour suggesting that if they want to do customizations they should create a new template.

Reply:
We can force a specific UUID while creating this template object but I didn't understood the use case.
It can be done in the following manner,
```python
default_template = Template(
    pk=uuid.UUID('66b1a809-2d83-4e33-8b2d-64aa5d4036a9'),  # This UUID would then be common for monitoring templates across openwisp instances
    ...
```
Let me know if the above is what you were suggesting and I remain unsure on how we can use the UUID of templates to upgrade them (so if possible a brief explanation for it would be helpful).

Reply:
@nepython the lua scripts are evolving rapidly and will need to be updated over time so it would be good to be able to update the lua code.

But users could customize that code so we should let them know that if they want to do that, they should create another template (so we do not overwrite their changes).